Cargo.toml

```diff
@@ -11,12 +11,24 @@ version = "0.1.0"
 [dependencies]
 rand = "0.4"
 bit-vec = "0.4.4"
+ff = "0.4"
 futures = "0.1"
 futures-cpupool = "0.1"
+group = "0.1"
 num_cpus = "1"
 crossbeam = "0.3"
-pairing = "0.14"
 byteorder = "1"
 
+[dependencies.pairing]
+git = "https://github.com/str4d/pairing"
```

Comment:
Is the idea that this will change back once https://github.com/zkcrypto/pairing/pull/91 has been merged?

Reply:
Yep, that is what happened.